packages/relay/src/TxStoreManager.ts

```diff
@@ -78,35 +78,31 @@ export class TxStoreManager {
     return await this.txstore.asyncFindOne({ txId: txId.toLowerCase() })
   }
 
-  async getTxsUntilNonce (signer: PrefixedHexString, nonce: number): Promise<StoredTransaction[]> {
-    return await this.txstore.asyncFind({
+  async getTxsInNonceRange (signer: PrefixedHexString, fromNonce: number, toNonce: number): Promise<StoredTransaction[]> {
```

Comment:
toNonce can be optional: setting $gte to "undefined" is supposed to mean not checking it.